_templates/hello.html

```diff
@@ -24,6 +24,9 @@
       {{ tagline }}
   </div>
 
+  <div class="Social">
+    {{ Social_Media }}
```

Comment:
```suggestion
    @{{ Social_Media }}
```
this will make it show the `@` symbol next to your username on the rendered site.

Reply:
if you want this to **link** to your site, you make the link here. 

Hints: 
- what is the HTML element that creates a link? 
- the URL for the `afro.tech` acct I showed in class is: `https://www.instagram.com/afro.tech/`
- the `{{ Social_Media }}` is a variable that sphinx uses. you can use it anywhere and as many times as you want